source/vir/src/sst_to_air.rs

```diff
@@ -285,6 +289,8 @@ pub(crate) fn exp_to_expr(ctx: &Ctx, exp: &Exp) -> Expr {
                         BinaryOp::Mul => panic!("internal error"),
                         BinaryOp::EuclideanDiv => air::ast::BinaryOp::EuclideanDiv,
                         BinaryOp::EuclideanMod => air::ast::BinaryOp::EuclideanMod,
+                        BinaryOp::BitXor => panic!("internal error"),
```

Comment:
Can we expand the "internal error" into something more detailed, so the person who sees it has a bit more context about what went wrong?

Reply:
Oh this one is should be an unreachable error, because we just had a clause that handles the translation. 
https://github.com/secure-foundations/verus/blob/e8058d8fe32c79c83e172722fddd338fa4627f24/source/vir/src/sst_to_air.rs#L273